toolkit/themes/shared/design-system/config.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint-env node */
+
+const StyleDictionary = require("style-dictionary");
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+/**
+ * CSS formatter that converts tokens that have a light and dark value
+ * to a light-dark() function. If we try to use the built-in CSS formatter,
+ * we will lose the "darkValue" or any other property that isn't "value".
+ */
+function lightDarkFormatter(args) {
+  let dictionary = args.dictionary;
+  let modifiedDictionary = dictionary.allTokens.map(token => {
+    // the `dictionary` object now has `usesReference()` and
+    // `getReferences()` methods. `usesReference()` will return true if
+    // the value has a reference in it. `getReferences()` will return
+    // an array of references to the whole tokens so that you can access their
+    // names or any other attributes.
+    if (token.original.dark) {
+      // Note: make sure to use `token.original.value` because
+      // `token.value` is already resolved at this point.
+      const lightRefs = dictionary.getReferences(token.original.value);
+      const darkRefs = dictionary.getReferences(token.original.dark);
+      if (lightRefs.length > 0 || darkRefs.length > 0) {
+        for (let i = 0; i < darkRefs.length; i++) {
+          let currentLightName = lightRefs[i]?.name;
+          let currentDarkName = darkRefs[i]?.name;
+          let lightValue = currentLightName ? 
+          `var(--${currentLightName})` : `${token.original.value}`;
+          let darkValue = currentDarkName ?
+          `var(--${currentDarkName})` : `${token.original.dark}`;
+          token.value = `light-dark(${lightValue}, ${darkValue})`
+        }
+      } else {
+        value = `light-dark(${token.original.value}, ${token.original.dark})`;
+        token.value = value;
```

Comment:
```suggestion
        token.value = `light-dark(${token.original.value}, ${token.original.dark})`;
```
Looks like `value` is no longer defined above so we either need to do `let value =`, or just make this a one liner